eng/referenceFromRuntime.targets

```diff
@@ -111,6 +111,15 @@
     <Error Condition="'@(_missingReferenceFromRuntime)' != ''"
            Text="Could not resolve ReferenceFromRuntime item(s) '%(_missingReferenceFromRuntime.OriginalReferenceFromRuntime)' from '$(RuntimeProjectFile)'." />
   </Target>
+
+  <Target Name="SetGenApiProperties"
+          BeforeTargets="GenerateReferenceAssemblySource"
+          Condition="'@(ReferenceFromRuntime)' != ''">
+    <PropertyGroup>
+      <GenAPILibPath>$(RuntimePath.Trim('\/'));$(CoreCLRArtifactsPath.Trim('\/'))</GenAPILibPath>
```

Comment:
Why is this needed?  I would expect any references needed to already be passed via https://github.com/dotnet/arcade/blob/a8ee27e1afd19641e279207ac9117a4116afdd87/src/Microsoft.DotNet.GenAPI/build/Microsoft.DotNet.GenAPI.targets#L80-L82

Reply:
ReferencePath ItemGroup is not populated as GenerateReferenceAssemblySource does not depend upon resolve assembly reference. I have a pr up for that https://github.com/dotnet/arcade/pull/5419